.env.dist

```diff
@@ -0,0 +1,3 @@
+CLIENT_ID= # set during the application is created
+CLIENT_SECRET= # create in the "Credentials" page of the application configuration
+TENANT_ID= # id of the Visma ERP company. Corresponds to `ipp-company-id` header for VNI
```

Comment:
I'm not sure about this comment. I was not able to find how to get "tenant ID" in the documentations, but in examples it looks like guid 
> af1140c1-52e0-46c7-b684-df894d4b8a5a

Reply:
UPD: found answer here 
https://community.visma.com/t5/Knowledge-base-in-Developers/Testing-the-Visma-net-ERP-API-from-Swagger/ta-p/560244